komga-webui/src/locales/en.json

```diff
@@ -166,12 +165,12 @@
   },
   "browse_readlist": {
     "edit_elements": "Edit elements",
-    "edit_readlist": "Edit read list"
+    "edit_readlist": "Edit reading list"
   },
   "browse_series": {
-    "earliest_year_from_release_dates": "This is the earliest year from the release dates from all books in the series",
-    "series_no_summary": "This series has no summary, so we picked one for you!",
-    "summary_from_book": "Summary from book {number}:"
+    "earliest_year_from_release_dates": "Earliest year of any released book in this series.",
```

Comment:
"any released book" is very different from "any book with a release date", and seems awkward

Reply:
Didn't think of that